content/post/115-protocol-labs-and-igalia-collaborating-on-dweb-in-browsers.md

```diff
@@ -0,0 +1,194 @@
+---
+date: 2021-01-15
+title: Protocol Labs and Igalia collaborating on dweb in browsers
+author: Frédéric Wang & Dietrich Ayala
+tags: ipfs, igalia, web platform, protocol handler, chromium, webkit, firefox, browsers
+header_image: ipfs-in-browsers.png
+snippet: Protocol Labs and Igalia started a collaboration to improve web platform support in Chromium, Firefox, and WebKit and help the distributed web community. This blog post provides an overview of the 2020 accomplishments, as well as future work.
+---
+
+## TL;DR
+
+* [Protocol Labs](https://protocol.ai/) and [Igalia](https://www.igalia.com/) started a collaboration that will continue next year.
+* [Distributed web schemes have been safelisted in Chrome 86](https://www.chromestatus.com/feature/4776602869170176)’s implementation of [custom handlers](https://html.spec.whatwg.org/multipage/system-state.html#custom-handlers) and [registered at IANA](https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml).
+* Chrome 89 will allow browser extensions to register cross-origin handlers or handlers for schemes with prefix `ext+`. Refinement is pending for the [permission UI](https://bugs.chromium.org/p/chromium/issues/detail?id=1079333).
+* Firefox 84 has improved support for [loading locally delivered mixed-resources](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content#Loading_locally_delivered_mixed-resources). Patches have also been submitted to WebKit but are pending on reviews and discussions.
+* Work is in progress to improve Chromium’s consistency and specification compliance regarding the notion of [secure contexts](https://w3c.github.io/webappsec-secure-contexts/), including removing non-standard [localhost](https://chromestatus.com/feature/5698580851458048) [names](https://chromestatus.com/feature/5668106045227008).
+* Miscellaneous other fixes have landed for the Firefox and Chromium’s implementations of custom handlers.
+
+## Background
+
+Nowadays, the majority of pages on the Web are coming from central servers controlled by their owners. [Protocol Labs](https://protocol.ai/) is developing the [IPFS protocol](https://ipfs.io) in order to shape a future Web which would instead be based on the idea of peer-to-peer infrastructures. There have been web platform and browser efforts to reach the [goal of a distributed Web](https://arewedistributedyet.com/).
```

Comment:
s/which would instead be/which could also support

... or something to that effect.  I realize this is minor seeming nit but "instead" is adding a lot of weight here unnecessarily.